src/utils/davixuri.cpp

```diff
@@ -233,7 +233,7 @@ void Uri::setProtocol(const std::string & protocol) {
 }
 
 void Uri::httpizeProtocol() {
-    if(d_ptr->proto == "s3" || d_ptr->proto == "dav" || d_ptr->proto == "gcloud" || d_ptr->proto == "swift") {
+    if(d_ptr->proto == "s3" || d_ptr->proto == "dav" || d_ptr->proto == "gcloud" || d_ptr->proto == "swift" || d_ptr->proto == "cs3") {
         setProtocol("http");
     }
     if(d_ptr->proto == "s3s" || d_ptr->proto == "davs" || d_ptr->proto == "gclouds" || d_ptr->proto == "swifts") {
```

Comment:
Would we have Reva tokens over `https`?
If so, shouldn't also `d_ptr->proto == "cs3s"` be added?

Reply:
I'd turn around this to another question: what is the meaning of say `s3s` vs. `s3`? I understood that the `s` suffix means secure vs. plain/unsecure (`https` over `http`, SSL-encrypted `rfios` over plain `rfio`, etc.), but `s3` or `gcloud` ARE `https` only, right? what's the purpose of defining both prefixes if only the secure one is to be used?

Likewise, `cs3` is GRPC over `https` and there's no option to have unsecure `cs3`.

Reply:
> but `s3` or `gcloud` ARE `https` only, right?

That's not necessarily the case. I've seen occurrences of `s3://` calls hanging, as the actual request was sent over `http://` and not `https://`.

For Reva, the translation from `cs3://` might always be to `https://`, in which case my proposed change is not needed.

Reply:
OK, after checking around it turns out we may have Reva running over `http` (though not for general production), therefore your proposed change is indeed fully relevant. In fact, Reva itself is typically deployed to listen to `http` and a reverse proxy is deployed in front for `https` (K8s, nginx, ...).

Given all that, let's go for `cs3` -> `http` and `cs3s` as an extra prefix resolved to `https`. @dynamic-entropy can you please add the `cs3s` prefix?

Reply:
Done